### PR TITLE
Transaction payments work via new `fungible` traits implementation

### PR DESCRIPTION
**Pull request by muharem**
Fixes: https://github.com/polkadot-fellows/runtimes/issues/328

Transaction payments work via new `fungible` traits implementation

### CODE REVIEW

File: system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -295,7 +296,7 @@ parameter_types! {
 impl pallet_transaction_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type OnChargeTransaction =
-		pallet_transaction_payment::CurrencyAdapter<Balances, DealWithFees<Runtime>>;
+		pallet_transaction_payment::FungibleAdapter<Balances, ResolveTo<StakingPot, Balances>>;
```
Comment (acatangiu) [inline system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs:299]: This PR changes the tx fees for some of the system parachains from `DealWithFees` (80% treasury, 20% block author) to `StakingPot` (100% goes to collators iiuc).

First, I want to raise visibility of this to make sure it's what we want (I haven't seen any convos on the topic so I don't know what's the general consensus).
Second, let's make it visible in the CHANGELOG.

**Review by acatangiu — APPROVED** [review]

File: system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs (hunk 1 above)
Comment (muharem) [reply system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs:299]: No I wanna only change it from currency to fungible. parachains_common::DealWithFees deposits everyting to staking pot.
https://github.com/paritytech/polkadot-sdk/blob/89b67bc69e5b8cdad49c378f6db0a48873548d35/cumulus/parachains/common/src/impls.rs#L62

Comment (muharem) [reply system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs:299]: Yes, they have a similar name, it can be confusing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Staking runtime api to check if reward is pending for an era ([polkadot-fellows/runtimes#318](https://github.com/polkadot-fellows/runtimes/pull/318))
 - Allow any parachain to have bidirectional channel with any system parachains ([polkadot-fellows/runtimes#329](https://github.com/polkadot-fellows/runtimes/pull/329))
 
+### Changed
+- Transaction payments work via new `fungible` traits implementation ([polkadot-fellows/runtimes#318](https://github.com/polkadot-fellows/runtimes/pull/332))
```
Comment (joepetrowski) [inline CHANGELOG.md:15]:
```suggestion

- Transaction payments work via new `fungible` trait implementation ([polkadot-fellows/runtimes#332](https://github.com/polkadot-fellows/runtimes/pull/332))
```

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**